apis/apps/v1alpha1/backupschedule_types.go

```diff
@@ -20,30 +20,56 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// BackupConditionType represents a valid condition of a Backup.
+type ScheduledBackupConditionType string
+
+const (
+	// ScheduledBackupPending means the scheduled backup job is pending, waiting for creation of the backup cronjob
+	ScheduledBackupPending ScheduledBackupConditionType = "Pending"
+	// ScheduledBackupScheduled means the scheduled backup cronjob was created successfully and no active backup jobs are running
+	// if there was an active backup job, the job has executed successfully and the backup data has been loaded into the nebula cluster.
+	ScheduledBackupScheduled ScheduledBackupConditionType = "Scheduled"
+	// ScheduledBackupRunning means there's an active backup job current running.
+	ScheduledBackupRunning ScheduledBackupConditionType = "Running"
+	// ScheduledBackupPaused means the schedule backup is currently suspended
+	ScheduledBackupPaused ScheduledBackupConditionType = "Paused"
+	// ScheduledBackupJobFailed means the active backup job has failed to execute successfully
+	ScheduledBackupJobFailed ScheduledBackupConditionType = "Backup job failed"
+	// BackupFailed means the backup cron job creation has failed.
+	ScheduledBackupFailed ScheduledBackupConditionType = "Cron Creation Failed"
```

Comment:
How about unify these to "Failed"

Reply:
@MegaByte875 I kept these separate so we know whether it is the NebulaScheduledBackup itself that failed (ScheduledBackupFailed) or the backup job it triggered (ScheduledBackupJobFailed), but we can unify these if that makes more sense.

Reply:
Fixed